packages/next-image-plus/src/background-image.tsx

```diff
@@ -56,16 +56,20 @@ export function getBackgroundImageProps(
 ): BackgroundImageData {
   const props: BackgroundImageData = {};
 
-  const mediaQueries = [];
-  for (const { breakpoint, media, url } of options) {
-    mediaQueries.push({
-      uuid: `${breakpoint}-${url}`,
-      media: media,
+  let mediaQueriesFinal = null;
+  // We only need media queries if there's more than 1 image.
+  if (options.length > 1) {
+    const mediaQueries = [];
+    for (const { breakpoint, media, url } of options) {
```

Comment:
Check if media is null and throw error:

Background image with src "${src}" is missing required "media" property. Either use a single background image, or add a media query.